.eslintrc.json

```diff
@@ -12,6 +12,13 @@
         "airbnb/hooks",
         "prettier"
     ],
+    "settings": {
```

Comment:
Followed this stackoverflow since I was getting unresolved path lint errors such as `Unable to resolve path to module './constants'`

https://stackoverflow.com/questions/55198502/using-eslint-with-typescript-unable-to-resolve-path-to-module

TLDR: We already have `eslint-plugin-import` package installed but we need to add the settings config for it as listed here in the doc and in the stackoverflow link: https://github.com/import-js/eslint-plugin-import#importextensions

Reply:
Good find! Thanks! :D